packages/react-google-maps-api/src/utils/helper.ts

```diff
@@ -6,8 +6,9 @@ const applyUpdaterToNextProps = (
   updaterMap: any,
   prevProps: any,
   nextProps: any,
+  // eslint-disable-next-line @getify/proper-arrows/params
```

Comment:
first of all I think we should remove the params length rule. 
second, easiest way to solve this error is to make this a regular function as it should be

Reply:
I agree, but I'm not sure how. As I understand, this eslint plugin evolves, and this rules a bit more complex. By now this comments absolutely fine.

Reply:
```
function applyUpdaterToNextProps(
  updaterMap: any,
  prevProps: any,
  nextProps: any,
  instance: any
): any {
  let map: any = {}

  const iter = (fn: any, key: string) => {
    const nextValue = nextProps[key]

    if (nextValue !== prevProps[key]) {
      map[key] = nextValue
      fn(instance, nextValue)
    }
  }

  forEach(updaterMap, iter)

  return map
}
```

this removes the error